server/auth.ts

```
@@ -1,64 +1,63 @@
+): void => {
   const token = req.cookies.token as JWTtoken
-  if (token && req.body.author) {
-    const author: IndexSignature<Author> = req.body.author
-    let decripted
-
-    Logger.info('req.body.author: ' + JSON.stringify(req.body.author))
+  if (!token || !req.body.author) {
+    Logger.info('Access from unexped route')
+    Logger.info('token: ' + token)
     Logger.info('req.cookies.token: ' + req.cookies.token)
+    return next(createError(403, 'unauthorized'))
```

Biome (1.9.4)

[error] The function should not return a value because its return type is void.

The function is here:

'void' signals the absence of value. The returned value is likely to be ignored by the caller.

(lint/correctness/noVoidTypeReturn)